src/portal_to_terra_input_from_anaset.py

```diff
@@ -0,0 +1,298 @@
+import urllib.parse
+from igvf_and_terra_api_tools import get_igvf_auth_and_api
+import os
+import pandas as pd
+import itertools
+import urllib
+from datetime import datetime
+import re
+
+# TODO:
+# 1) Update the script to reflect the read_name (no more guessing)
+
+# def get_igvf_auth_and_api(igvf_site: str = 'production'):
+#     """Set up IGVF data portal access and set up IGVF python client api.
+#     """
+#     if igvf_site == 'production':
+#         host = 'https://api.data.igvf.org'
+#     elif igvf_site == 'sandbox':
+#         host = 'https://api.sandbox.igvf.org'
+#     config = Configuration(
+#         access_key=os.environ["IGVF_API_KEY"],
+#         secret_access_key=os.environ["IGVF_SECRET_KEY"],
+#         host=host
+#     )
+#     client = ApiClient(config)
+#     return IgvfApi(client)
+
+
+#  HACK: A map that translates an assay term to either ATAC or RNA
+# assay_terms_map = {
+#     '/assay-terms/OBI_0003109/': 'rna',
+#     '/assay-terms/OBI_0002762/': 'atac',
+#     '/assay-terms/OBI_0002764/': 'atac',
+#     '/assay-terms/OBI_0002631/': 'rna',
+#     '/assay-terms/OBI_0003090/': 'rna',     # For test analysis set only
+#     '/assay-terms/OBI_0003089/': 'atac'     # For test analysis set only
+# }
+
+# HACK: A map that uses platform_preferred-assay-title_assay-type to inference read map for pipelines.
+read_type_by_platform_map = {
+    '/platform-terms/EFO_0008637/_SHARE-seq_atac': {'R1': 'read1', 'R2': 'read2'},
```

Comment:
I found the keys here a little confusing because you don't use the platform names, maybe they could be comments or something,.